app/javascript/shared/components/tags.jsx

```diff
@@ -87,10 +80,31 @@ class Tags extends Component {
       this.state.cursorIdx < this.textArea.value.length &&
       this.textArea.value.length < this.state.prevLen + 1
     ) {
-      this.textArea.selectionStart = this.textArea.selectionEnd = this.state.cursorIdx;
+      this.textArea.selectionEnd = this.state.cursorIdx;
+      this.textArea.selectionStart = this.textArea.selectionEnd;
     }
   }
 
+  // getter methods must be above render and below componentDidUpdate
```

Comment:
this is probably not needed, since we have eslint to enforce it

Reply:
I added that comment because that is what the eslint error message said. I think I actually copy and pasted the message itself for that comment. I will remove the comment though! It was mostly for me so I could remember why I did it. I forgot to remove it before committing.